cruise-control/src/test/java/com/linkedin/kafka/cruisecontrol/executor/ExecutorTest.java

```diff
@@ -231,6 +231,58 @@ public void testTimeoutLeaderActions() throws InterruptedException {
     waitUntilExecutionFinishes(executor);
   }
 
+  @Test
+  public void testForceStopExecution() throws InterruptedException {
```

Comment:
Running this test locally took more than 10seconds in my local test env. I noticed that it is using `Thread.sleep()`. Can we avoid using it in unit test?

Reply:
All the tests in `ExecutorTest` are slow mainly due to the slowness of setting up a kafka cluster(and zookeeper).  

Here the `Thread.sleep(10);` is used in a while loop when we are waiting for executor to reach `INTER_BROKER_REPLICA_MOVEMENT_TASK_IN_PROGRESS` state and it is because we do not want to free burning CPU cycle waiting waiting Cruise Control executing. the time unit here is `millisecond` so it would not block unnecessarily long.

Reply:
I am curious if we can reduce the overall time by combining the cluster setup of multiple tests -- i.e. do we need to setup the cluster from scratch for each individual test case or can we do it once for selected test cases?

Reply:
There will be one issue if we pursue that way. 

Several tests in `ExecutorTest` will execute some partition movement/leadership movement tasks, If they share the same Kafka cluster under the hood, the tests will intervene each other.  It will be hard to implement/debug the tests because the cluster state at the beginning of test execution is changing.  And we will also observe different behavior between running specific test alone and running the suite.